src/Form/Form.js

```diff
@@ -14,11 +15,15 @@ class Form {
         this.isSignup = this.formAnalyzer.isSignup
         this.Device = DeviceInterface
         this.attachTooltip = DeviceInterface.attachTooltip
-        this.allInputs = new Set()
-        this.emailNewInputs = new Set()
-        this.credentialsInputs = new Set()
-        this.creditCardInputs = new Set()
-        this.unknownInputs = new Set()
+
+        /** @type Object<'all' | SupportedMainTypes, Set> */
+        this.inputs = {
+            all: new Set(),
+            emailNew: new Set(),
+            credentials: new Set(),
+            creditCard: new Set(),
+            unknown: new Set()
+        }
```

Comment:
This was suggested in the previous PR review. I chose to keep the `all` set there for convenience when we loop through all fields.